templates/kube-apiserver.yaml

```diff
@@ -21,6 +21,12 @@ spec:
         - --secure-port={{kube_apiserver_secure_port}}
         - --advertise-address={{ansible_default_ipv4.address}}
         - --admission-control={{kube_admission_control}}
+{% if enable_external_admission_controller_webhook %}
+        - --runtime-config=admissionregistration.k8s.io/v1alpha1
+        - --proxy-client-cert-file={{kube_cert_dir}}/client.pem
```

Comment:
I'm OK with this change, but don't see a reason why we can't just re-use apiserver.pem and apiserver-key.pem here.

Reply:
Because the keys are signed differently. Key Encipherment is enabled on the apiserver.pem which prevents them from being used to identify as an ssl client.

Reply:
Adding 'Digital signature' keyUsage should make it work for ssl clients. But I'm not sure how we'd add that to existing certificates. So I'm OK with this change.